src/macports1.0/restore.tcl

```diff
@@ -84,22 +80,6 @@ namespace eval restore {
         return 0
     }
 
-    proc check_port_command {} {
```

Comment:
I had the same doubt of running restore after upgrading. This clears things up a bit.

But restore serves another use case where you want to restore a different snapshot and not migrate. This check was kept, keeping in mind that it’s just better to check the port command and notify the user in case, the user needs a migration.